emukit/examples/gp_bayesian_optimization/optimization_loops.py

```diff
@@ -14,13 +14,15 @@
 
 def create_bayesian_optimization_loop(x_init: np.ndarray, y_init: np.ndarray, parameter_space: ParameterSpace,
                                       acquisition_type: AcquisitionType, model_type: ModelType,
+                                      c_init: np.ndarray = None,
```

Comment:
Let's be more expressive
```suggestion
                                      cost_init: np.ndarray = None,
```

Reply:
good point